pkg/container-collection/containers.go

```diff
@@ -50,6 +50,9 @@ type Container struct {
 	// See https://github.com/opencontainers/runtime-spec/blob/main/bundle.md
 	Bundle string `json:"bundle,omitempty"`
 
+	// SandboxId is the sandbox id for the corresponding pod
+	SandboxId string `json:"sandboxId,omitempty"`
```

Comment:
For now, this information will be only available for containers that are created when IG is already running and only for containerd and crio, right? Please update the comment. BTW, isn't this info provided by the runtime?

Reply:
> For now, this information will be only available for containers that are created when IG is already running and only for containerd and crio, right?
>
Correct.

> Please update the comment.
> 
We need to document who enriches what somewhere else, not here.

> BTW, isn't this info provided by the runtime?
Yes, it is. However, implementing this is not worth it because we only use 'SandboxID' to retrieve the PodLabels for new CRI-O containers from the runtime enricher. Why only in this case? [The `runtimeClient.GetContainer` call](https://github.com/inspektor-gadget/inspektor-gadget/blob/f02743d64d81a01f3212bc0a9e9387e67a3e9ec7/pkg/container-collection/options.go#L76) will fail because the container doesn't exist yet in CRI-O. However, the SandboxPods do exist. So, we can at least enrich the container with the pod labels.